packages/backend/src/index.ts

```diff
@@ -62,10 +58,10 @@ function makeCreateEnv(config: Config) {
       database,
       cache,
       config,
-      reader,
       discovery,
       tokenManager,
       scheduler,
+      reader: undefined as any,
```

Comment:
reader has been removed from backend-common. this is just a temporary hack to get CI to pass so that we can move to the next PR where we switch to the new backend